### DISCUSSION TITLE
Are container IDs deterministic?

### DISCUSSION THREAD

**Discussion by TheMarstonConnell** [gerbera/gerbera#3054] [Q&A] [answered]
I'm building a music metadata platform and pairing gerbera with a database for external metadata. My question is, can I rely on the container ID for albums or will they change at some point? Thanks

**Comment by KarlStraussberger** [comment]
No, the container id will never be reused. If you delete it and reimport it, you will get a new id

**Reply by TheMarstonConnell** [reply]
So if I was to shutdown the server, or restart the device, as long as I don't reimport the files, the IDs stay the same?

**Reply by KarlStraussberger** [reply, marked as answer]
Yes it does.
